_content/development.md

```diff
@@ -0,0 +1,48 @@
+# Web application development
+
+## Web and mobile
+
+We specialise in developing software applications for both Web and mobile using *JavaScript*, *Node.js* and *React*.
```

Comment:
Raises the question of whether we expect the current cohort to be able to develop in React even though it's not on the syllabus this time around (i.e. just pick it up in their own time)